apps/src/p5lab/AnimationTab/DeleteAnimationDialog.jsx

```diff
@@ -20,8 +20,6 @@ export default class DeleteAnimationDialog extends React.Component {
         assetType = i18n.animation();
         break;
       case P5LabType.SPRITELAB:
```

Comment:
Just curious, why don't we need to set `assetType` for `P5LabType.SPRITELAB`? I don't see a default value so just wondering what would happen in that case.

Reply:
ope, just saw the `Additional refactor for DeleteAnimationDialog` in the description - I'll take a look at that previous PR!

Reply:
Yeah, this is something I just learned you could do with switch statements. You don't actually need the context of the other PR to understand it either. Because the middle case doesn't ever `break;` it's going to do whatever happens in the following case. Effectively, this it's kind of like we're saying `this.props.LabType == P5LabType.SPRITELAB || this.props.labType == P5LabType.POETRY`. It's called [fall-through0(https://en.wikipedia.org/wiki/Switch_statement#Fallthrough) if you want to know more.

Reply:
> I realized I could have taken advantage of fallthrough in switch statements

After taking a look at those changes I'm not sure I totally understand what happens in the `P5LabType.SPRITELAB`. Does it fall through to a different switch case?

Reply:
oooohh I see! thanks for explaining that!